optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -385,7 +385,7 @@ def pre_attn_forward(
                 if flash_attention_causal_mask:
                     # causal masking on first token requires inputs to be of the same lenght
                     with ht.sdp_kernel(enable_recompute=flash_attention_recompute):
-                        attn_output = FusedSDPA.apply(query_states, key_states, value_states, None, 0.0, True, None)
+                        attn_output = FusedSDPA.apply(query_states, key_states, value_states, None, 0.0, True, None , "fast")
```

Comment:
can we enable "fast softmax" using a flag in generation_config similar to what is done for flash_attention? This will enable us to control this easily without going into modeling file.

Reply:
https://github.com/wszczurekhabana/optimum-habana-fork/commit/85f159a84551176e24ea47216debe0c30da11861#diff-30aeee6868dd1de34878aca0583f57bb5b0dd9a2a8511a80e9a6b2645f39ce6bR310

We can use something similar to what's being done for inference.